dashboard/test/mailers/previews/pd_fit_weekend1819_registration_mailer_preview.rb

```diff
@@ -15,7 +15,9 @@ def build_registration(status)
     user = build :teacher, email: 'facilitator@code.org'
     application = build :pd_facilitator1819_application, user: user
 
-    workshop = application.find_fit_workshop(course: 'CS Discoveries', city: 'Phoenix', year: 2018)
+    workshop = application.find_fit_workshop(course: 'CS Discoveries', city: 'Phoenix', year: 2018) ||
+      create(:pd_workshop, course: Pd::Workshop::COURSE_CSD, subject: Pd::Workshop::SUBJECT_CSD_TEACHER_CON, processed_location: {city: 'Phoenix'}, num_sessions: 5)
+
```

Comment:
nit: let's use the constant `Pd::Workshop::COURSE_CSD` (or include Pd::WorkshopConstants and just refer to `COURSE_CSD`) on line 18 too, rather than the string.